src/fluree/db/query/exec/where.cljc

```diff
@@ -34,8 +34,14 @@
        (assoc ::meta m))))
 
 (defn match-iri
-  [mch iri]
-  (assoc mch ::iri iri))
+  ([iri]
```

Comment:
Thanks for adding this arity and the function below. Both good additions to this api.